### PR TITLE
Emoji 15

### PR DESCRIPTION
**Pull request by willchavez**
**Added**
- **Shaking face** / 3D, Color, Flat, High Contrast
- **Pink heart** / 3D, Color, Flat, High Contrast
- **Light blue heart** / 3D, Color, Flat, High Contrast
- **Grey heart** / 3D, Color, Flat, High Contrast
- **Leftwards pushing hand** / 3D, Color, Flat, High Contrast
- **Rightwards pushing hand** / 3D, Color, Flat, High Contrast
- **Moose** / 3D, Color, Flat, High Contrast
- **Donkey** / 3D, Color, Flat, High Contrast
- **Wing** / 3D, Color, Flat, High Contrast
- **Blackbird** / 3D, Color, Flat, High Contrast
- **Goose** / 3D, Color, Flat, High Contrast
- **Jellyfish** / 3D, Color, Flat, High Contrast
- **Hyacinth** / 3D, Color, Flat, High Contrast
- **Ginger root** / 3D, Color, Flat, High Contrast
- **Pea pod** / 3D, Color, Flat, High Contrast
- **Folding hand fan** / 3D, Color, Flat, High Contrast
- **Hair pick** / 3D, Color, Flat, High Contrast
- **Maracas** / 3D, Color, Flat, High Contrast
- **Flute** / 3D, Color, Flat, High Contrast
- **Khanda** / 3D, Color, Flat, High Contrast
- **Wireless** / 3D, Color, Flat, High Contrast

### CODE REVIEW

**Review by seangibbz — APPROVED** [review]
Pose of Donkey emoji is a bit awkward, but does match similar platform fonts (i.e. Google Noto).  Other than that, all designs are quite exquisite.

**Comment by Lamparter** [comment]
> Pose of Donkey emoji is a bit awkward, but does match similar platform fonts (i.e. Google Noto). Other than that, all designs are quite exquisite.

@seangibbz These are the emoji as they appear on Windows, it is not up to @willchavez to change the design.

**Comment by DellZHackintosh** [comment]
@willchavez Please take a look on Shaking Face (3D Variant).

It looks like that image lost a face, but it's supposed to have 3 faces.

File: assets/Shaking face/3D/shaking_face_3d.png
[the remark this answers is not in the document]
Comment (DellZHackintosh) [reply assets/Shaking face/3D/shaking_face_3d.png:1]: Ok, it looks great now.

**Review by DellZHackintosh — APPROVED** [review]

**Review by spencer-nelson — APPROVED** [review]

**Pull request merged**